balboa-http/src/main/scala/com/socrata/balboa/server/MetricsServlet.scala

```diff
@@ -67,28 +59,23 @@ class MetricsServlet extends JacksonJsonServlet
       return unacceptable.result
     }
 
-    val begin = System.currentTimeMillis()
-
     val range = DateRange.create(period, ServiceUtils.parseDate(date).getOrElse {
       contentType = json
       return malformedDate(date).result
     })
 
-    try
-    {
-      val iter = dataStore.find(entityId, period, range.start, range.end)
-      var metrics = Metrics.summarize(iter)
+    timer("period queries") {
+      try {
```

Comment:
I don't think you need this try block any more, do you? No catch clause, no finally.